lpcompiler.pas

```diff
@@ -1767,6 +1798,9 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
         Exit;
       end;
 
+      if (Tokenizer.Tok in [tk_kw_Deprecated, tk_kw_Experimental, tk_kw_UnImplemented]) then
```

Comment:
Either this should be changed to a while loop, or TLapeType_Method should store an enum instead of booleans.